dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -124,10 +146,11 @@ export default defineComponent({
       if (this.lksgDataAndMetaInfo.length) {
         this.lksgDataAndMetaInfo.forEach((oneLksgDataset: DataAndMetaInformationLksgData) => {
           const dataIdOfLksgDataset = oneLksgDataset.metaInfo?.dataId ?? "";
-          const dataDateOfLksgDataset = oneLksgDataset.data.social?.general?.dataDate ?? "";
-          this.listOfDataDateToDisplayAsColumns.push({
+          // const dataDateOfLksgDataset = oneLksgDataset.data.social?.general?.dataDate ?? "";
```

Comment:
What about this commented out code?

Reply:
It's not necessary anymore because the column identifiers now consist of objects with the fileds "dataId" and "reportingPeriod".
I deleted it, and I also delted it for SfdrPanel.vue